eventstore/whatsapp_actions.py

```diff
@@ -78,6 +79,30 @@ def handle_event(event):
     if event.is_hsm_error:
         handle_whatsapp_hsm_error(event)
 
+    if event.fallback_channel is True:
+        handle_fallback_delivery_error(event)
+
+
+def handle_fallback_delivery_error(event):
+    try:
+        df = DeliveryFailure.objects.get(contact_id=event.recipient_id)
+        df.number_of_failures += 1
```

Comment:
This might cause a race condition when two processes are trying to increment this at the same time. Using `F` expressions, we can delegate this to the database instead, ie.
```
df.number_of_failures = F("number_of_failures") + 1
df.save(update_fields="number_of_failures")
df.refresh_from_db()
```